src/main/java/jewellery/inventory/repository/UserRepository.java

```diff
@@ -8,7 +8,7 @@
 
 @Repository
 public interface UserRepository extends JpaRepository<User, UUID> {
-  Optional<User> findByName(String name);
+  Optional<User> findByFirstName(String name);
```

Comment:
I don't know if findByName was used anywhere but it seems findByFirstName is not used anywhere now.